Source/JavaScriptCore/runtime/ConsoleClient.h

```diff
@@ -80,7 +80,7 @@ class ConsoleClient : public CanMakeWeakPtr<ConsoleClient> {
     virtual void screenshot(JSGlobalObject*, Ref<Inspector::ScriptArguments>&&) = 0;
 
 private:
-    enum ArgumentRequirement { ArgumentRequired, ArgumentNotRequired };
+    enum class ArgumentRequirement { ArgumentRequired, ArgumentNotRequired };
```

Comment:
In a case like this we should change the enumeration values since they are repetitive once we switch to an `enum class`.
```suggestion
    enum class ArgumentRequirement { Required, NotRequired };
```
Or possibly even shorter, something like this.
```suggestion
    enum class ArgumentRequirement { No, Yes };
```

Reply:
Fixed with `Yes` and `No`.